src/dune_rules/dune_file.mli

```diff
@@ -182,7 +182,10 @@ module Library : sig
   val main_module_name : t -> Lib_info.Main_module_name.t
 
   val to_lib_info :
-    t -> dir:Path.Build.t -> lib_config:Lib_config.t -> Lib_info.local
+       t
+    -> dir:Path.Build.t
+    -> lib_config:Lib_config.t
+    -> Lib_info.local Memo.Build.t
```

Comment:
Why do we need this change?

Reply:
Because `Blang.eval` is now done in the `Memo.Build` monad, and so evaluating `enabled_if` introduces the monad